community/detector/cves/CVE-2021-29441/src/main/java/com/google/tsunami/plugins/detectors/cves/cve202129441/CVE202129441DetectorBootstrapModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.cves.cve202129441;
+
+import com.google.tsunami.plugin.PluginBootstrapModule;
+
+/**
+ * An example Guice module that bootstraps the {@link CVE202129441VulnDetector}.
+ */
+public final class CVE202129441DetectorBootstrapModule extends PluginBootstrapModule {
+
+	@Override
```

Comment:
[Google Java Code Style](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation): Block indentation: +2 spaces

You can use `google-java-format` to reformats your PR to comply with Google Java Style: https://github.com/google/google-java-format